src/main.js

```diff
@@ -227,6 +227,11 @@ function dispatch(event) {
   }
 }
 
+// 判断 element 是否已经绑定事件
+function isElementBind (element) {
+  return elementHasBindEvent.some(ele => ele.isSameNode(element))
```

Comment:
[Node​.isSame​Node
](https://developer.mozilla.org/zh-CN/docs/Web/API/Node/isSameNode) This feature is not compatible with most browsers.